author.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
+
+class Author < Item
+  attr_accessor :id, :name, :items
+
+  def initialize(name)
+    super()
+    @id = rand(1...9999)
+    @name = name
+    @items = []
+  end
+
+  def add_item(_item)
+    @items << items
```

Comment:
- [ ] 

- [ ] In the add_item method, when adding an item to the @items array, you should use the _item parameter passed to the method, not the undefined variable items. So, it should be @items << _item instead of @items << items.

Reply:
Done